common-issues.md

```diff
@@ -204,5 +204,5 @@ See also: [#587](https://github.com/wp-cli/wp-cli/issues/587)
 
 The `wp core update` command is designed to work for WordPress 3.4 and above. To be able to update an older website to latest WordPress, you could try one of the following alternatives:
 
-1. **Fully-automated:** Run `wp core download --force` to download latest WordPress and replace it with your files (don't worry, `wp-config.php` will remain intact). Then, run `wp core update-db` to update the database. Since the procedure isn't ideal, run once again `wp core download --force` and the new version should be available.
+1. **Fully-automated:** Run `wp core download --force` to download the latest WordPress and replace it with your files (don't worry, `wp-config.php` will remain intact). Then, run `wp core update-db` to update the database. Since the procedure isn't ideal, run once again `wp core download --force` and the new version should be available.
```

Comment:
This is grammatically wrong, the original sentence was correct. Please revert this change.

Reply:
@schlessera Done